src/transform.ts

```diff
@@ -24,7 +24,7 @@ function transformType(resource: Resource, data: unknown) {
     }
 }
 
-function transformJsonData(json: unknown) {
+export function transformJsonData(json: unknown) {
```

Comment:
We should discuss what is the most "best practise" way of doing this for our case. Testing only the publicly available interface or also testing the internals, i.e. by exposing them through `export` keyword.

Some interesting thoughts and ideas on this topic: https://stackoverflow.com/a/54116345